torch/csrc/jit/script/compiler.h

```diff
@@ -17,6 +17,14 @@ struct CallsiteDescriptor {
   bool allow_varargs;
 };
 
+struct DefAndTypes {
```

Comment:
nit: maybe `DefAndTypes -> AnnotatedDef`. The problem with naming something based on what it contains is that you have to rename it whenever you add fields.